governance.qmd

```diff
@@ -0,0 +1,120 @@
+---
+title: "R-Multiverse Governance"
+---
+
+The R-Multiverse project is run in a collaborative and open way.
```

Comment:
As Noam mentioned today, it would be good to expand on this section to include:

1. The mission, values, and purpose of R-multiverse.
2. The scope and purpose of this governance document.